### ISSUE TITLE
[MLS Spec change] Better name for preconfigured sender ID (#598)

### ISSUE THREAD

**Issue by raphaelrobert** [openmls/openmls#1025]
**Link to the exact changes**
https://github.com/mlswg/mls-protocol/pull/598

**Description of the changes**
> Fixes #588

**Comment by duesee** [comment]
The change is basically ...

```diff
struct {
    SenderType sender_type;
    switch (sender_type) {
        case member:        LeafNodeRef member;
-       case preconfigured: opaque external_key_id<V>;
+       case preconfigured: opaque sender_id<V>;
        case new_member:    struct{};
    }
} Sender;
```
... but the `Sender::Preconfigured` [does not use](https://github.com/openmls/openmls/blob/4b6f8a4eb17e9bea4d522772c7308a7dcbc84199/openmls/src/framing/sender.rs#L17) this attribute name.

I thought about adding the presentation language already but it makes probably more sense to do it when working on https://github.com/openmls/openmls/issues/949.

There is nothing to do here, really.

**Comment by duesee** [comment]
@franziskuskiefer Can we close this?

**Comment by raphaelrobert** [comment]
#949 super-seedes this and should be addressed in #902
